core/codegen/src/derive/from_form.rs

```diff
@@ -84,7 +84,7 @@ pub fn derive_from_form(input: TokenStream) -> TokenStream {
         .try_map_fields(move |_, fields| {
             let (constructors, matchers, builders) = fields.iter().map(|field| {
                 let (ident, span) = (&field.ident, field.span().into());
-                let default_name = ident.as_ref().expect("named").to_string();
+                let default_name = crate::Ident::from(ident.as_ref().expect("named").clone()).to_string();
```

Comment:
I don't think we need to convert this to `crate::Ident` since we just call `to_string`.

Reply:
This is a long way to get to `ident.unraw()`, which is all that is really needed here.